.vscode/launch.json

```diff
@@ -1,16 +1,60 @@
 {
-  // Use IntelliSense to learn about possible attributes.
-  // Hover to view descriptions of existing attributes.
-  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-  "version": "0.2.0",
-  "configurations": [
-    {
-      "name": "Create extensions",
-      "type": "go",
-      "request": "launch",
-      "mode": "auto",
-      "program": ".",
-      "args": ["create", "testdata/extension.config.yml"]
-    }
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
```

Comment:
Does this project include documentation for contributors? I'd consider moving turning some steps from the PR description into documentation for contributors.